test/sleep.js

```diff
@@ -0,0 +1,3 @@
+export default function sleep(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
```

Comment:
this could be a `.ts` as well while we are at it

Reply:
Done 👍